src/savegame_json.cpp

```diff
@@ -3542,6 +3542,21 @@ void submap::store( JsonOut &jsout ) const
     }
     jsout.end_array();
 
+    jsout.member( "crafting_bills" );
+    jsout.start_array();
+    for( auto &elem : crafting_bills ) {
+        jsout.write( elem.first.x );
```

Comment:
```suggestion
        jsout.write( elem.first );
```
The JSON system knows how to write a tripoint. Also change the code in the deserialization function accordingly.